packages/react/src/drawer-title/DrawerTitle.tsx

```diff
@@ -0,0 +1,37 @@
+import * as RadixDrawer from "@radix-ui/react-dialog";
+import { forwardRef } from "react";
+
+import { type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { Heading } from "../heading";
+import { extractSprinkles } from "../sprinkles";
+import { Text } from "../text";
+
+type DrawerTitleProps = BoxProps<
+  typeof RadixDrawer.Title,
+  {
+    description?: string;
+  }
+>;
+
+export const DrawerTitle = forwardRef<HTMLDivElement, DrawerTitleProps>(
+  ({ children, description, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    return (
+      <Flex gap="xs" pb="md" pt="lg" px="lg" {...sprinkleProps}>
+        <Heading asChild level="4">
+          <RadixDrawer.Title ref={ref} {...restProps}>
```

Comment:
adding `ref` and `restProps` to the `Title` doesn't feel right - let's add the ref and other props to the outer component instead:
```suggestion
      <Flex gap="xs" pb="md" pt="lg" px="lg" ref={ref} {...props}>
        <Heading asChild level="4">
          <RadixDrawer.Title>
```

Reply:
let's do the same thing on `DialogTitle` as well